pkgs/apps/xtb-iff/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "xtb-iff";
+  version = "1.1";
+
+  src = fetchurl {
+    url = "https://github.com/grimme-lab/xtbiff/releases/download/v${version}/xtbiff.tar.xz";
+    hash = "sha256-bV3uELo57LHiY3S3v4LiNk2OvtW281zc5M7XMkXClvU=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    tar -xvf ${src}
```

Comment:
`stdenv` can not handle xz compression?

Reply:
It can but it will complain that no source directory is produced. It is really just a single, statically linked executable in tar archive ... For some reasons ...